decisions/wc_undead_decisions.txt

```diff
@@ -148,6 +148,7 @@ decisions = {
 				trait = creature_plague_eruptor
 				trait = creature_abomination
 				trait = creature_lich
+				trait = creature_wicker_golem
```

Comment:
So, `creature_wicker_behemoth` can become golems?

Reply:
Basically: evolve from stage 2 to stage 3.

Reply:
The problem is that the stage 2 is better than the stage 3.

Reply:
> So, `creature_wicker_behemoth` can become golems?

@Velius94 are you sure it's resolved?